### PR TITLE
Tmedia 444 block shell pt 2

### PR DESCRIPTION
**Pull request by JackHowa**
## Description
Setup basic identity block with features, sources, etc 

## Jira Ticket
- [TMEDIA-444](https://arcpublishing.atlassian.net/secure/RapidBoard.jspa?rapidView=875&modal=detail&selectedIssue=TMEDIA-444&assignee=5e387d6a1b1d910e5dfd7a9b)

## Acceptance Criteria

- [x] should have a package.json, npmignore, many features available
- [x] published to at least canary tag
- [x] dependency should include "@arc-publishing/sdk-identity": "^1.55.0"
- [x] Should export the useIdentity hook
- [x] A test should pass

## Test Steps

1. Checkout this branch `git checkout TMEDIA-444-block-shell-pt-2`
2. Checkout the branch in fusion-news-theme for `git checkout TMEDIA-400-subs-poc`. Make sure it's the most updated. version of this branch too. update the apiOrigin as before in the dropbox docs 
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/subscriptions-block,@wpmedia/identity-block`
3. Add a header position login within the header-nav-chain-block

Screen Shot 2021-08-10 at 15 17 48
Screen Shot 2021-08-10 at 15 17 20

### Before

No features available for identity 

### After

Show feature for login identity 

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
- [x] Confirmed all the test steps a reviewer will follow above are working.
- [x] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [x] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [x] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [ ] Confirmed this PR has unit test files
  - [ ] Ran `npm run test`, made sure all tests are passing
  - [ ] If the amount of work to write unit tests for this change are excessive,
please explain why (so that we can fix it whenever it gets refactored).
- [x] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

### CODE REVIEW

File: blocks/identity-block/features/header-account-action/default.jsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  }, [Identity, loggedIn]);
+
+  const handleLogout = () => {
+    Identity.logout().then(() => { setIsLoggedIn(false); setUser(null); });
+  };
+
+  if (!isInitialized) {
+    return null;
+  }
+
```
Comment (JackHowa) [inline blocks/identity-block/features/header-account-action/default.jsx:50]: @farleyawapo in this logic block, it should say, if logged in, then render that account button

File: blocks/identity-block/features/header-account-action/styles.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    height: 2rem;
+    padding: 0 0.5rem;
+    text-decoration: none;
+
+    &:hover {
+      color: #fff;
+      border: 1px solid #fff;
+    }
+  }
+}
```
Comment (JackHowa) [inline blocks/identity-block/features/header-account-action/styles.scss:23]: @farleyawapo styles for account button can be written here

Comment (JackHowa) [reply blocks/identity-block/features/header-account-action/styles.scss:23]: @farleyawapo if mobile responsive breakpoint, then hide the "account" text via styles

File: blocks/identity-block/components/Identity.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+// import { useFusionContext } from 'fusion:context';
+// import getProperties from 'fusion:properties';
+// import { isServerSide } from '@wpmedia/engine-theme-sdk';
+
+const useIdentity = () => ({
+  Identity,
+  isInitialized: true,
+});
+
+// keep skelton for future use
```
Comment (mattkim93) [inline blocks/identity-block/components/Identity.jsx:12]: I've no issue with uncommenting this, if you're ok with it

**Review by mattkim93 — APPROVED** [review]
LGTM, thanks

File: blocks/identity-block/components/Identity.jsx (hunk 3 above)
Comment (JackHowa) [reply blocks/identity-block/components/Identity.jsx:12]: will uncomment when the next ticket comes in -- good idea

**Pull request merged**